bazel/get_workspace_status

```diff
@@ -12,6 +12,16 @@
 # If the script exits with non-zero code, it's considered as a failure
 # and the output will be discarded.
 
+# If this SOURCE_VERSION file exists then it must have been placed here by a
+# distribution doing a non-git, source build.
+# Distributions would be expected to echo the commit/tag as BUILD_SCM_REVISION
+if [ -f SOURCE_VERSION ]
+then
+    echo "BUILD_SCM_REVISION $(cat SOURCE_VERSION)"
+    echo "BUILD_SCM_STATUS Distribution"
```

Comment:
Not sure if it's worth complicated the SCM status parsing to add this third value, but don't feel that strongly either.